.github/workflows/prod-alt.yml

```
@@ -0,0 +1,32 @@
+      prerelease: false
+      branch: main-alt
+    secrets: inherit
+
+  deploy:
+    needs: [static-checks, release]
+    uses: ./.github/workflows/deploy.yml
+    with:
+      environment: production-alt
+      branch: main-alt
+      version: ${{ needs.release.outputs.version }}
+    secrets: inherit
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)